### DISCUSSION TITLE
allow for more headings like h3 - h6

### DISCUSSION THREAD

**Discussion by fpigeonjr** [vslinko/obsidian-zoom#101] [Archive]
Would it be possible to get zoom into headings deeper than heading two?
Maybe allow for h3, h4, h5, and h6? Although it is rare for me to go much deeper than an h3.

Love this plugin...keep up the great work.

**Comment by vslinko** [comment]
Hello @fpigeonjr 

Obsidian Zoom already allows you to zoom h3-h6 headers. 
If something is not working, please report the issue.
<img width="764" alt="Screenshot 2023-08-02 at 10 58 36" src="https://github.com/vslinko/obsidian-zoom/assets/205823/77b510c2-01c0-4352-906b-a346357875a8">

**Comment by fpigeonjr** [comment]
Confirmed, it is working for me. Will close this feature request. 
Thanks for the quick response.
